### PR TITLE
delete bot-link and add pr check link CI

### PR DESCRIPTION
**Pull request by nowinkeyy**
### Summary
:link::memo::heavy_check_mark:

This pull request removes an unused workflow file and adds a new workflow file and a configuration file for checking broken links in markdown files. The new workflow uses the `lychee-action` and the `lychee.toml` file to run a link checker on pull requests.

> _`lychee-action` checks_
> _broken links in markdown files_
> _autumn leaves fall fast_

### Walkthrough
* Add a GitHub Actions workflow to check for broken links in markdown files ([link](https://github.com/labring/sealos/pull/3277/files?diff=unified&w=0#diff-3a1211724ffea638f3e41811cc8655c2fd92e4db104113a138ae78d2510fd2e7L1-R15), [link](https://github.com/labring/sealos/pull/3277/files?diff=unified&w=0#diff-0df195050d656faa60917eccaef12ba9df2185cf3533d0e6070f0713a43809f1R6-R7))
* Delete an unused workflow file `bot-link.yml` ([link](https://github.com/labring/sealos/pull/3277/files?diff=unified&w=0#diff-fd39904cb398940466b0a7d8f1850b2cd120684b013d26e70c236081a1dd88c9))

### CODE REVIEW

**Comment by nowinkeyy** [comment]
@cuisongliu plz review this pr

**Comment by cuisongliu** [comment]
https://github.com/labring/sealos/actions/runs/5173245415/jobs/9318337968?pr=3277 


It should be an error

**Comment by nowinkeyy** [comment]
I changed the config path to see if I could read it.

File: .github/workflows/check-link-error.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.7.0
+        with:
+          args: --config ../../lychee.toml
```
Comment (cuisongliu) [inline .github/workflows/check-link-error.yml:17]: https://github.com/lycheeverse/lychee-action#alternative-approach

**Pull request closed without merging**